winlogbeat/Makefile

```diff
@@ -12,7 +12,3 @@ gen:
 # This is called by the beats packer before building starts
 .PHONY: before-build
 before-build:
-	# Windows
-	sed -i 's|#\{0,1\}\(to_files:\).*|\1 true|' $(PREFIX)/$(BEATNAME)-win.yml
-	sed -i 's|#\{0,1\}\(level:\).*|\1 info|' $(PREFIX)/$(BEATNAME)-win.yml
-	sed -i '/log files/{n;s|#\{0,1\}path:.*|path: C:/ProgramData/winlogbeat/Logs|}' $(PREFIX)/$(BEATNAME)-win.yml
```

Comment:
Yeah, I thought so, but that makes it inconsistent with the rest of the Beats. The reason we keep it to ERR is that WARN is too verbose in Packetbeat, maybe in other beats as well. Maybe we should move most of those WARNs to DEBUG and then switch to INFO by default.

I think our logging system, in general, is due for refactoring.